pom.xml

```diff
@@ -38,7 +38,7 @@
     </parent>
 
     <properties>
-      <java.version>1.8</java.version>
```

Comment:
Since the docker image is built on top of 11, I changed this version as well.

Reply:
sounds good - I don't think we have a use case anymore for supporting java 8